server/constants/routes.go

```diff
@@ -21,13 +21,15 @@ const (
 	PathPipelineRequest           = "/pipeline_request"
 
 	// Azure API paths
-	CreateTask               = "/%s/%s/_apis/wit/workitems/$%s?api-version=7.1-preview.3"
-	GetTask                  = "%s/%s/_apis/wit/workitems/%s?api-version=7.1-preview.3"
-	GetPullRequest           = "%s/%s/_apis/git/pullrequests/%s?api-version=6.0"
-	GetGitRepositories       = "%s/%s/_apis/git/repositories?api-version=6.0"
-	GetGitRepositoryBranches = "%s/%s/_apis/git/repositories/%s/refs?filter=heads"
-	PipelineApproveRequest   = "%s/%s/_apis/release/approvals/%d?api-version=6.0"
-	GetProject               = "/%s/_apis/projects/%s?api-version=7.1-preview.4"
-	CreateSubscription       = "/%s/_apis/hooks/subscriptions?api-version=6.0"
-	DeleteSubscription       = "/%s/_apis/hooks/subscriptions/%s?api-version=6.0"
+	CreateTask                              = "/%s/%s/_apis/wit/workitems/$%s?api-version=7.1-preview.3"
+	GetTask                                 = "%s/%s/_apis/wit/workitems/%s?api-version=7.1-preview.3"
+	GetPullRequest                          = "%s/%s/_apis/git/pullrequests/%s?api-version=6.0"
+	GetGitRepositories                      = "%s/%s/_apis/git/repositories?api-version=6.0"
+	GetGitRepositoryBranches                = "%s/%s/_apis/git/repositories/%s/refs?filter=heads"
+	PipelineApproveRequest                  = "%s/%s/_apis/release/approvals/%d?api-version=6.0"
+	GetProject                              = "/%s/_apis/projects/%s?api-version=7.1-preview.4"
+	CreateSubscription                      = "/%s/_apis/hooks/subscriptions?api-version=6.0"
+	DeleteSubscription                      = "/%s/_apis/hooks/subscriptions/%s?api-version=6.0"
+	PathGetSubscriptionFilterPossibleValues = "/subscriptions/filters"
```

Comment:
I think this is a plugin API path and the rest are Azure API paths

Reply:
yes that was there due to resolving merge conflicts by Raghav